install.sh

```diff
@@ -91,12 +120,7 @@ function install() {
             sudo systemctl stop "$SERVICE"
         fi
         echo "creating '/etc/systemd/system/$SERVICE$SERVICE_EXTENSION'"
-        sudo cp -f "$SERVICES_DIR/$SERVICE$SERVICE_EXTENSION" "/etc/systemd/system/$SERVICE$SERVICE_EXTENSION" > "/dev/null"
-        sudo systemctl daemon-reload
-        echo "enabling [$SERVICE]"
-        sudo systemctl enable "$SERVICE"
-        echo "starting [$SERVICE]"
-        sudo systemctl start "$SERVICE"
+        cat "$SERVICES_DIR/$SERVICE$SERVICE_EXTENSION" | sed -e "s/%DIRECTORY%/${INSTALL_DIRECTORY//\//\\/}/" | sudo tee "/etc/systemd/system/$SERVICE$SERVICE_EXTENSION" > "/dev/null"
```

Comment:
[Calling `sudo` during packaging is not a good idea either](https://bbs.archlinux.org/viewtopic.php?id=295558), another one of the effects of makepkg using fakeroot to limit where the script gets effective root-like permissions within a sandbox. In the current state this would in fact spam the package build with error messages like `ERROR: ld.so: object 'libfakeroot.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.` masking the actual issue, and that's if sudo is even allowed to run arbitrary commands which is not the case on hardened installations.

You also already point out in the README that one needs to run the install script with sudo anyways for it to work properly.

I'd suggest [adding a check if the necessary root access was given](https://stackoverflow.com/a/18216122) to make sure the user is following the README properly, and removing the then redundant `sudo` calls throughout the script.